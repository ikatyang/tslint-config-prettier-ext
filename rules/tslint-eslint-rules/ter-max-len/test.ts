this.is.a.long().long().long().long().long().long().long().long().long().long().method();
