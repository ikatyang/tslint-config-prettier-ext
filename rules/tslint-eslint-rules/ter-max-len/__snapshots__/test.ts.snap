// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should affect error message after formatting 1`] = `
"
<<<<<< before
ERROR: (ter-max-len) /rules/tslint-eslint-rules/ter-max-len/test.ts[1, 1]: Line 1 exceeds the maximum line length of 80.

======
no error

>>>>>> after
"
`;

exports[`should be pretty after formatting 1`] = `
"
<<<<<< before
this.is.a.long().long().long().long().long().long().long().long().long().long().method();

======
this.is.a
  .long()
  .long()
  .long()
  .long()
  .long()
  .long()
  .long()
  .long()
  .long()
  .long()
  .method();

>>>>>> after
"
`;
